lib/manageiq/rpm_build/build_hotfix.rb

```diff
@@ -0,0 +1,75 @@
+require 'awesome_spawn'
+require 'fileutils'
+require 'pathname'
+require 'yaml'
+
+module ManageIQ
+  module RPMBuild
+    class BuildHotfix
+      include Helper
+
+      attr_reader :rpm_spec
+
+      def generate_rpm
+        where_am_i
+
+        FileUtils.mkdir_p(BUILD_DIR)
```

Comment:
Should this be:

```suggestion
        FileUtils.mkdir_p(HOTFIX_DIR)
```

If not, and it's assumed HOTFIX_DIR will exist, then I don't know why you'd need the mkdir_p

Reply:
Removed